lib/widget/cliprect.cpp

```diff
@@ -39,10 +39,8 @@ void ClipRectWidget::run(W_CONTEXT *psContext)
 
 bool ClipRectWidget::processClickRecursive(W_CONTEXT *psContext, WIDGET_KEY key, bool wasPressed)
 {
-	W_CONTEXT newContext;
-	newContext.xOffset = psContext->xOffset;
-	newContext.yOffset = psContext->yOffset;
-	newContext.mx = psContext->mx;
+	W_CONTEXT newContext(psContext);
+	newContext.yOffset = psContext->yOffset - offset.y;
```

Comment:
I think we should keep the `xOffset` here. It is not currently used anywhere, but in theory the `ClipRectWidget` could be used to clip horizontally too.

I'd say, either keep the `xOffset` or remove it completely, and keep only the `yOffset`.